integration_tests/test_get_requests.py

```diff
@@ -61,3 +63,12 @@ def test_trailing_slash(session):
 
     r = requests.get("http://localhost:8080/trailing/")
     assert r.text == "Trailing slash test successful!"
+
+
+@pytest.mark.benchmark
+@pytest.mark.parametrize("key, value", [("fakesession", "fake-cookie-session-value")])
```

Comment:
@VishnuSanal , could you please do an e2e test, where you create a new endpoint and check a response?